### PR TITLE
Feat/labels archive and trash pages

### PR DESCRIPTION
**Pull request by tonymike99**
In this PR, I have added the labels, archive and trash page

The reason there are 1000+ lines is because I didn't merge the previous branch to dev.. So please ignore the number of lines

Please check only these files

### CODE REVIEW

**Review by purvasharmadev — APPROVED** [review]
code is readable, good job with naming, good to go.

File: src/pages/Trash/Trash.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  // SET DOCUMENT TITLE
+  useDocumentTitle("Trash");
+
+  // ****************************************************************************************************
+
+  const { trash, getTrash } = useTrash();
+
+  useEffect(() => {
+    getTrash();
+  }, []);
```
Comment (sudipkundu999) [inline src/pages/Trash/Trash.jsx:16]: I see this repeating on every page. Instead, make a useEffect like this in the header or some context and then do all the functional calls together

**Review by sudipkundu999 — APPROVED** [review]
The code looks good. Good to merge

File: src/pages/Trash/Trash.jsx (hunk 1 above)
Comment (tonymike99) [reply src/pages/Trash/Trash.jsx:16]: Okay Sudip. Thanks for pointing it out. Will make the changes

**Pull request merged**